hypothesis-python/src/hypothesis/vendor/pretty.py

```diff
@@ -123,30 +136,35 @@ class RepresentationPrinter:
 
     """
 
-    def __init__(self, output=None, *, context=None):
+    def __init__(
+        self,
+        output: Optional[TextIOBase] = None,
+        *,
+        context: Optional["BuildContext"] = None,
+    ) -> None:
         """Pass the output stream, and optionally the current build context.
```

Comment:
```suggestion
        """Optionally pass the output stream and the current build context.
```
(older than this PR, but oops)